src/Template/Element/sentences/sentence_icons.ctp

```diff
@@ -0,0 +1,27 @@
+<?php
+$sentenceBaseUrl = $this->Url->build([
+    'controller' => 'sentences',
+    'action' => 'show',
+]);
+
+$responsiveNgClass = '{activated: translation.showActions, deactivated: !translation.showActions}';
+?>
+
+<div class="indicator" ng-if="<?= $angularVar ?>.correctness === -1">
+    <md-icon class="md-warn" >warning</md-icon>
+    <md-tooltip md-direction="top">
+        <?= __('This sentence is not reliable.') ?>
+    </md-tooltip>
+</div>
+
+<md-button class="md-icon-button" ng-class="<?= $responsiveNgClass ?>" ngclipboard data-clipboard-text="{{<?= $angularVar ?>.text}}">
+    <md-icon>content_copy</md-icon>
+    <md-tooltip><?= __('Copy sentence') ?></md-tooltip>
+</md-button>
+
+<?= $this->element('sentence_buttons/audio', ['angularVar' => $angularVar]); ?>
+
+<md-button class="md-icon-button" ng-class="<?= $responsiveNgClass ?>" href="<?= $sentenceBaseUrl ?>/{{<?= $angularVar ?>.id}}">
```

Comment:
Since you use interpolation markup (`{{}}`) it would be safer to use `ng-href` here.